### PR TITLE
Workflow BUG Fix

### PR DESCRIPTION
**Pull request by Shikhar15606**
## NA

NA

Closes: NA

### Fixed a workflow Bug

Checking all the files for linting and formatting errors

## BUG FIX

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement in the existing features
- [ ] Other

## Mention any unusual behaviour of your code (Write NA if not)
NA

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.

## Additional Info (optional)
NA

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,12438 @@
+Arguments: 
+  /usr/local/bin/node /usr/share/yarn/bin/yarn.js check-linting
+
+PATH:
```
Comment (Namanl2001) [inline yarn-error.log:4]: This file seems unintentional

Comment (Shikhar15606) [reply yarn-error.log:4]: Yes it is unintentional, Let me remove it

**Review by Namanl2001 — APPROVED** [review]
LGTM! let's resolve conflicts before merging

**Comment by Shikhar15606** [comment]
Merge Conflict Resolved

**Pull request merged**